calico/openstack/test/test_plugin_etcd.py

```diff
@@ -95,14 +109,23 @@ def assertEtcdDeletes(self, expected):
             self.assertEqual(self.recent_deletes, expected)
         self.recent_deletes = set()
 
-    def etcd_read(self, key, recursive=False, timeout=None):
+    def etcd_read(self, key, wait=False, waitIndex=None, recursive=False, timeout=None):##add wait&waitIndex
```

Comment:
Two notes:

1. Python convention for variable names is lowercase_with_underscores, not camelCase. (Exceptions and classes use PascalCase.) The variables throughout this function should be renamed accordingly.
2. There's an extra comment on the end of this line, but it doesn't seem to add anything. Please remove it.